src/store/Entity/Module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from 'vuex';
```

Comment:
This file is the entry point to the entity module in a way. The common convention is to call it `index.ts`.